osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -243,6 +243,9 @@ public override bool OnExiting(IScreen next)
             this.FadeOut(150);
             cancelLoad();
 
+            if (Background != null)
+                Background.UpdateDim.Value = false;
```

Comment:
a single class shouldn't set dim to disabled when it never enabled it. if this is because player enabled updatedim, then shouldn't player disable it?

there's two ways you can look at it:

- If player enables in enter, it should disable in exit. (apply to every usage)
- Always set enabled or disabled in OnEntering and OnResuming in every class that has a beatmap background. (no screen undoes its applications, but it's always correct because every screen applies).

Reply:
this is actually some left over code from when I was testing a bug, shouldn't be necessary here.